Documentation/boostrap-new-os.md

```diff
@@ -0,0 +1,140 @@
+TODO: describe how to re-crossgen the managed assemblies
+## Building nuget packages
+Now that the bootstrap CLI works, it can be used to build coreclr, corefx and core-setup repos in its completeness, including the managed code. First set the `DOTNET_TOOL_DIR` env variable to the path where the boostrap CLI is located. That will cause the build process to use these tools instead of trying to download a `.tar.gz` package with OS specific tools from Azure.
+You can use the boostrap tools to build any recent state of the three repos. `release/2.0.0` branch can be used to build the 2.0.0 release packages or `master` branch to build the latest stuff.
+For the sake of simplicity, the following paragraphs assume the following:
+* we are building release configuration
+* we are targetting x64
+* the new RID is `mynewrid`
+* the coreclr, corefx and core-setup folders with clones of the three repos are in the same directory
+### Adding support for the new RID
+#### CoreCLR
+* Unless the target OS has `/etc/os-release` file that can be used to extract the RID, the `initHostDistroRid` function in the `build.sh` file needs to be updated to be able to detect the target OS and report the proper RID by setting the `__HostDistroRid` to the RID combined with the target architecture (e.g. `mynewrid-x64`).
+* In `src/.nuget/dir.props` add a new `OfficialBuildRID` element with the new RID and target architecture. So for example, `<OfficialBuildRID Include="mynewrid-x64" />`
+#### CoreFX
+* Unless the target OS has `/etc/os-release` file that can be used to extract the RID, the `initHostDistroRid` function in the `src/Native/build-native.sh` file needs to be updated to be able to detect the target OS and report the proper RID by setting the `__HostDistroRid` to the RID combined with the target architecture (e.g. `mynewrid-x64`).
+* In `dependencies.props`, the `CoreClrPackageVersion` needs to be updated to contain the version of packages built in the coreclr repo.
+* In `external/runtime/runtime.depproj`, the content of the `Version` element under `<PackageReference Include="Microsoft.NETCore.DotNetHost">` and `<PackageReference Include="Microsoft.NETCore.DotNetHostPolicy">` needs to be updated to contain the version of packages built in the core-setup repo.
+* In `external/test-runtime/XUnit.Runtime.depproj`, add `<NoWarn>NU1603</NoWarn>` to the first `<PropertyGroup>` element.
+* In `pkg/Microsoft.NETCore.Platforms/runtime.json`, add new elements describing the new RID and its relationship under the `runtimes` key. Here is an example of what we would add for Alpine Linux 3.6:
```

Comment:
@ericstj recently updated this process.  See https://github.com/dotnet/corefx/tree/master/pkg/Microsoft.NETCore.Platforms#adding-new-rids.  You no longer need to update the runtime.json file manually.  We generate it from the `runtimeGroups.props` file.